q2_fondue/entrezpy_clients/_pipelines.py

```diff
@@ -53,20 +49,20 @@ def _get_run_ids(
     # search for IDs
     es = samp_ids_pipeline.add_search(
         {'db': db, 'term': " OR ".join(ids)},
-        analyzer=ESearchAnalyzer(ids, log_level)
+        analyzer=ESearchAnalyzer(ids)
     )
     if elink:
         # given bioproject, find linked SRA runs
         el = samp_ids_pipeline.add_link(
             {'db': 'sra'},
-            analyzer=ELinkAnalyzer(), dependency=es
+            analyzer=ElinkAnalyzer(), dependency=es
```

Comment:
The `Elink` classes found in `entrezpy` have everything that is need by us and more so I'm removing our implementation completely.